keras/src/backend/openvino/numpy.py

```diff
@@ -764,7 +764,20 @@ def greater_equal(x1, x2):
 
 
 def hstack(xs):
-    raise NotImplementedError("`hstack` is not supported with openvino backend")
+    if not isinstance(xs, (list, tuple)):
+        raise TypeError("Input to `hstack` must be a list or tuple of tensors.")
+    if len(xs) == 0:
+        raise ValueError("Input list to `hstack` cannot be empty.")
```

Comment:
please remove, not needed checks. TF backend does not have such check for hstack implementation.